src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/vscode.proposed.d.ts

```diff
@@ -19,51 +19,6 @@ declare module 'vscode' {
 
     //#region Semantic tokens: https://github.com/microsoft/vscode/issues/86415
 
-    export class SemanticTokensLegend {
```

Comment:
All changes here and to `vscodeAdapter.ts` are just making sure our local "fake" API matches theirs.